build/debian/control

```diff
@@ -6,7 +6,7 @@ Build-Depends: debhelper (>= 9),
                pkg-config,
                docbook-to-man,
                libglu1-mesa-dev,
-               libqt4-opengl-dev | libqt4-dev (>= 4.6.0),
+               libqt5opengl5-dev | qt5-default (>= 5.2.0),
```

Comment:
Have you tested that Mixxx actually works with Qt 5.2?

Reply:
Travis uses Trusty which is on 5.2, so it builds at least.

Reply:
I have still some build issues on Trusty with Qt 5.2 a PR is on the way.